src/main/java/duke/command/AddCommand.java

```diff
@@ -0,0 +1,67 @@
+package duke.command;
+
+import java.util.Arrays;
+
+/**
+ * A Class that extends the Command class.
+ * It is specifically designed for a Command for adding.
+ *
+ * @author Gu Geng
+ */
+public class AddCommand extends Command {
+    private duke.task.Task task;
+
+    /**
+     * Returns a AddCommand object with the information provided.
+     *
+     * @param command A String containing information that can possibility be used to create an AddCommand object.
+     * @throws duke.DukeException Will be thrown if information provided are insufficient/incorrect.
+     */
+    public AddCommand(String... command) throws duke.DukeException {
```

Comment:
Perhaps it will be better formatting to use 
```
if (condition) {
    statements;
} else if (condition) {
    statements;
} else {
    statements;
}
``` 
instead of nested
```
if (condition) {
    statements;
} else {
    statements;
}
```